labs/b2.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Beginner Lab 2 - Core Shell
+layout: lab
+---
+
+Welcome to Lab 2! In this lab you will be learning how to work productively in a shell.
+
+Don't forget to use Google and `man` when stuck. The resources linked at the bottom may be helpful as well.
+
+![XKCD 1319](https://imgs.xkcd.com/comics/automation.png)
+
+## Setting up
+This lab requires a bash shell, vim, and tmux.
+If you do not have tmux: `apt-get tmux`
+
+## Pipes and Redirection
+Chaining together commands is essential to automating your way through the shell. Here's a quick cheat sheet:
+
+`>` : Save output to a file.
```

Comment:
I'd clarify that this is redirect stdout to a file, so you can naturally extend that explanation when introducing `2>` and `2>&1`